tests/helpers.go

```diff
@@ -0,0 +1,59 @@
+package tests
+
+import (
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+
+	. "github.com/onsi/ginkgo"
+)
+
+func ChangeDirectory(dir string) {
+	err := os.Chdir(dir)
+	if err != nil {
+		Fail("Failed to change directory: " + err.Error())
+	}
+}
+
+func CreateTemplate() string {
+	input := []byte("test-gin-templates")
+
+	reader, writer, err := os.Pipe()
+	if err != nil {
+		Fail("Failed to connected pair of files: " + err.Error())
```

Comment:
This error message is kind ow weird 😆 When does that happen and what error it is?

Reply:
Actually this command is for creating file to store stdin value.
Updated error message in ce7e2d0d9a1a7c4828c96a2db73ca20c70fad0f6